src/main/java/uk/gov/hmcts/reform/em/stitching/domain/BundleFolder.java

```diff
@@ -0,0 +1,89 @@
+package uk.gov.hmcts.reform.em.stitching.domain;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.List;
+
+@Entity
+@Table(name = "bundle_folder")
+public class BundleFolder extends AbstractAuditingEntity implements Serializable {
+
+
+//    Can I delete this?
+//    @OneToOne(mappedBy="bundle_folder")
+//    private DocumentTask documentTask;
+
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "sequenceGenerator")
+    @SequenceGenerator(name = "sequenceGenerator")
+    private Long id;
+
+    private String description;
+    private String folderName;
+    private String orderFoldersBy;
+    private String orderDocumentsBy;
+
+    @ElementCollection
+    private List<String> documents;
+
+    @ElementCollection
+    private List<String> folders;
```

Comment:
Wait, why does an individual `BundleFolder` have a list of folders

Reply:
The HLD has folders[] in the BundleFolder object, I'm guessing it's for subfoldering